### PR TITLE
Converted customisations module from Android to Java

### LINKED ISSUE #47: Convert 'customisations' module from Android to Java
The 'customisations' does not need to be an Android module, and should be converted to a Java module. Should we wish to support multiplatform in the future we will need these modules to not be Android specific

### PR DESCRIPTION
**Pull request by LachlanMcKee**
## Description
Fixes #47

## Check list

- [x] I have updated `CHANGELOG.md` if required.
- [x] I have updated documentation if required.

### CODE REVIEW

File: gradle/publication.gradle [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+    android {
+        publishing {
+            singleVariant('release') {
+                withSourcesJar()
+                withJavadocJar()
+            }
         }
     }
+} else {
+    java {
```
Comment (LachlanMcKee) [inline gradle/publication.gradle:14]: I've verified this works by using `publishToMavenLocal` and inspecting the output

File: gradle/publication.gradle [hunk 2]
```diff
@@ -1,20 +1,31 @@
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
 
-android {
-    publishing {
-        singleVariant('release') {
```
Comment (CherryPerry) [inline gradle/publication.gradle:6]: Can you name it `'java'`, so we do not need to add another split in `publishing` block?

Comment (LachlanMcKee) [reply gradle/publication.gradle:6]: @CherryPerry I don't believe so. This is the `release` variant of the library as opposed to `debug`. If I named it `java` it wouldn't find the build.

Also, I am improving the plugin in this PR: https://github.com/bumble-tech/appyx/pull/51

**Comment by CherryPerry** [comment]
I hope it won't break transitive dependency in RIBs.

**Comment by LachlanMcKee** [comment]
If this does cause an issue with RIBs, I will revert this. I am pretty confident it should not cause an issue however

**Pull request merged**